### PR TITLE
Drop 3.6 OSes support list

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:
- oVirt 3.6 has reached EOL, dropping it from OSes support list

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @sandrobonazzola

This pull request needs review by: @mykaul

### CODE REVIEW

File: source/download/index.html.haml [hunk 1]
```diff
@@ -52,9 +52,9 @@ wiki_warnings: list-item?
         ### Supported OSes (Engine)
 
         -   Fedora 23
-        -   CentOS Linux 6.8 (3.6 only), 7.2
-        -   Red Hat Enterprise Linux 6.8 (3.6 only), 7.2
-        -   Scientific Linux 6.7 (3.6 only), 7.2
+        -   CentOS Linux 7.2
+        -   Red Hat Enterprise Linux 7.2
+        -   Scientific Linux 7.2
```
Comment (mureinik) [inline source/download/index.html.haml]: Shouldn't all of these be "7.2 and above"?

Comment (sandrobonazzola) [inline source/download/index.html.haml]: @mureinik will be in a separate patch.
